distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.web.cfg

```diff
@@ -38,3 +38,15 @@ org.ops4j.pax.web.ssl.keypassword=${javax.net.ssl.keyStorePassword}
 
 # Client authenticate at server is wanted
 org.ops4j.pax.web.ssl.clientauthwanted=true
+
+#Included SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.included=${https.cipherSuites}
+
+#Excluded SSL/TLS Cipher Suites Regular Expressions
+org.ops4j.pax.web.ssl.ciphersuites.excluded=.*RC4.*,SSL.*,.*anon.*,.*3DES.*
+
+#Included SSL/TLS Protocols
+org.ops4j.pax.web.ssl.protocols.included=${https.protocols}
+
+#Excluded SSL/TLS Protocols
+org.ops4j.pax.web.ssl.protocols.excluded=SSLv3,TLSv1
```

Comment:
Having both, included and excluded, makes things a little confusing. What if a protocol is in both or in neither? Could we get this to work with either an exclusion or an inclusion list instead but not both?